### PR TITLE
Translate the "by" word

### PR DESCRIPTION
**Pull request by gergo85**

### CODE REVIEW

File: modules/cms/lang/en/lang.php [hunk 1]
```diff
@@ -77,7 +78,7 @@
         'export_folders_label' => 'Folders',
         'export_folders_comment' => 'Please select the theme folders you would like to export',
         'delete_button' => 'Delete',
-        'delete_confirm' => 'Are you sure you want to delete this theme? It cannot be undone!',
```
Comment (LukeTowers) [inline modules/cms/lang/en/lang.php]: Why did you change this?

Comment (gergo85) [inline modules/cms/lang/en/lang.php]: Because of: https://github.com/octobercms/october/commit/d6a7fdc74c8726149358545aff4e360a01bdf4fb#diff-73ae1299d2c352fefb63eb86c178f48dL114

Comment (LukeTowers) [inline modules/cms/lang/en/lang.php]: Perfect, thanks for the explanation!

File: modules/system/controllers/updates/_install_themes.htm [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h4 class="section-header">
                                 </div>
                                 <div class="details">
                                     <h4><?= $theme['name'] ?></h4>
-                                    <p>by <?= $theme['author'] ?></p>
+                                    <p><?= e(trans('cms::lang.theme.by')) ?> <?= $theme['author'] ?></p>
```
Comment (daftspunk) [inline modules/system/controllers/updates/_install_themes.htm]: Bad practice here. Some languages may include "by" after the author name. This should really be `by_author` with value `by :name`.

Comment (gergo85) [inline modules/system/controllers/updates/_install_themes.htm]: I changed the translations: #2849
